packages-backend/express/src/middlewares/session-middleware.spec.ts

```diff
@@ -195,7 +195,7 @@ describe.each`
 describe('when audience is missing', () => {
   it('should throw a validation error', () => {
     // @ts-ignore
-    expect(() => createSessionMiddleware({})).toThrowError(
+    expect(() => createSessionMiddleware({})).toThrow(
```

Comment:
Is this a mandatory change? 
I'm wondering if users updating Jest would also need to update their tests like this (and the other updates we needed to apply).
If that's so, maybe we could also mention it in the changelog.

Reply:
For reference, these changes are due to ESLint errors

<img width="692" alt="image" src="https://user-images.githubusercontent.com/1110551/233368542-28b6620e-5c13-4136-a427-b6840a5822e0.png">

https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-alias-methods.md